vuu-ui/cypress/e2e/layout-management/api/api.utils.ts

```diff
@@ -0,0 +1,116 @@
+import { ApplicationJSON, LayoutJSON } from "@finos/vuu-layout";
+import { LayoutMetadata, LayoutMetadataDto } from "@finos/vuu-shell";
+
+export type LayoutResponseDto = {
+  id: string;
+  definition: LayoutJSON;
+  metadata: LayoutMetadata;
+};
+
+export const LAYOUT_API_BASE_URL = "http://localhost:8081/api";
```

Comment:
I have dropped a couple of comments on the file that uses this const. If those changes are applied I think we don't need to export this